sith/settings.py

```diff
@@ -472,9 +472,11 @@
 SITH_PRODUCT_SUBSCRIPTION_TWO_SEMESTERS = 2
 SITH_PRODUCTTYPE_SUBSCRIPTION = 2
 
+# Defines which club lets its member the ability to make subscriptions
 SITH_CAN_CREATE_SUBSCRIPTIONS = [1]
 
-SITH_CAN_READ_SUBSCRIPTIONS = [85]
+# Defines which club lets its members the ability to see users subscription history
```

Comment:
Plutôt "list of club ids where member can read subscriptions of other users"

Reply:
Ouais il manque un `s` à club pour qu'on comprenne mais le commentaire est clair